Makefile

```diff
@@ -18,6 +18,7 @@ XGOIMAGE = statusteam/xgo:$(XGOVERSION)
 XGOIMAGEIOSSIM = statusteam/xgo-ios-simulator:$(XGOVERSION)
 
 DOCKER_IMAGE_NAME ?= status-go
+DOCKER_FILE_PATH ?= --file _assets/build/Dockerfile
```

Comment:
Has it been tested that building a docker image (`make docker-image`) still works as expected?